src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -148,6 +147,8 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
   override def adjustInputPaths(callKey: CallKey, inputs: CallInputs, workflowDescriptor: WorkflowDescriptor): CallInputs = inputs mapValues gcsPathToLocal
   override def adjustOutputPaths(call: Call, outputs: CallOutputs): CallOutputs = outputs mapValues gcsPathToLocal
 
+  def fileHasher: FileHasher = { wdlFile: WdlFile => getCrc32c(GoogleCloudStoragePath(wdlFile.value)) }
```

Comment:
Nothing is calling this, also it's using a different hash algorithm under the hood.